python/ct/client/log_client.py

```diff
@@ -197,8 +199,10 @@ def _parse_consistency_proof(response, servername):
 class RequestHandler(object):
     """HTTPS requests."""
     def __init__(self, connection_timeout=60, ca_bundle=True):
-      self._timeout = connection_timeout
-      self._ca_bundle = ca_bundle
+        if isinstance(ca_bundle, str):
```

Comment:
(1) the API doesn't make sense anymore and (2) the isinstance check counters Python's duck-typing.
I propose to set ca_bundle to None by default and just pass it in (should be OK according to module documentation: http://httplib2.googlecode.com/hg/doc/html/libhttplib2.html)